home/tests/test_content_import_export.py

```diff
@@ -711,13 +731,20 @@ def test_variations(self, impexp: ImportExportFixture) -> None:
             ]
 
         cp_imp_exp_wablks = [
-            WABlk("Message 1", next_prompt="Next message", variation_messages=m1vars),
+            WABlk(
+                "Message 1",
+                buttons=[{"type": "next_message", "value": {"title": "Next message"}}],
+                variation_messages=m1vars,
+            ),
             WABlk(
                 "Message 2, variable placeholders as well {{0}}",
-                next_prompt="Next message",
+                buttons=[{"type": "next_message", "value": {"title": "Next message"}}],
```

Comment:
Do we still have tests for `next_prompt` now that it's not gone completely? Do we support mixing buttons and `next_prompt` in the same message? (If not, we should probably enforce that somehow.)

Reply:
Hmm that's true. We do support mixing them (they should be treated as just separate things, and the application has to decide which one to use, to allow a smooth changeover), so I'll update this test to check that all combinations can work